src/Controller/Controller.php

```diff
@@ -170,6 +170,13 @@ class Controller implements EventListenerInterface, EventDispatcherInterface
      */
     protected $middlewares = [];
 
+    /**
+     * View classes for content negotiation.
+     *
+     * @var array<string>
```

Comment:
```suggestion
     * @var list<class-string>
```

Reply:
Using an associative array would potentially allow overriding a class. RequestHandler's `viewMap` uses an associative array.

Reply:
Ok, if that's an expected use case, works for me.